parsons/quickbooks/quickbooks.py

```diff
@@ -0,0 +1,734 @@
+from parsons import Table
+from parsons.utilities import check_env
+from parsons.utilities.api_connector import APIConnector
+import logging
+
+# Set up logger
+logger = logging.getLogger(__name__)
+
+QB_URI = "https://rest.tsheets.com/api/v1/"
+
+
+class QuickBooks:
+    """
+    Instantiate the QuickBooks class.
+
+    TODO: Add information about the difference between accounting vs time
```

Comment:
Not a huge deal, but would be nice not to publish the docs with a 'todo' in them. You can always link to a guide distinguishing the two if you don't want to have to write up the distinction yourself.